apps/charterafrica/src/components/Datasets/DatasetFilterBar.js

```diff
@@ -159,36 +81,37 @@ const DatasetFilterBar = React.forwardRef(function DatasetFilterBar(
                 }}
               />
             )}
+            value={sort}
           />
         </Grid>
         <Grid item xs={12} sm={4} lg={3} overflow="hidden">
-          <StyledAutocomplete
-            defaultValue={[labels.countries]}
+          <ComboBox
+            label={labels.countries}
             multiple
-            options={[labels.countries, ...countries]}
-            onChange={handleChangeCountry}
+            options={countriesOptions}
+            onChange={onChangeCountries}
             renderInput={(params) => <StyledAutocompleteInput {...params} />}
             renderTags={(checkedCountries, getTagProps) => (
               <StyledAutocompleteTags {...getTagProps} typography="p1">
                 {listToLabel(checkedCountries, "countries", 2)}
```

Comment:
To handle long country names
```suggestion
                {listToLabel(checkedCountries, "countries")}
```

Reply:
@kilemensi Yes. The above suggestion fixes it.